README.md

```diff
@@ -782,7 +782,7 @@ try {
 
 #### google/cloud-error-reporting
 
-Google Stackdriver Error Reporting can be installed separately by requiring the `google/cloud-error-reporting` composer package:
+[Google Stackdriver Error Reporting](https://github.com/GoogleCloudPlatform/google-cloud-php-error-reporting) can be installed separately by requiring the [`google/cloud-errorreporting`](https://packagist.org/packages/google/cloud-error-reporting) composer package:
```

Comment:
Link should be: https://github.com/GoogleCloudPlatform/google-cloud-php-errorreporting

Reply:
fixed